var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -66,7 +66,8 @@ class NeurodamusCore(SimModel):
 
     depends_on('python@2.7:', type=('build', 'run'))
 
-    mech_name = "neurodamus"
+    # Dont apply name for now for compat with neuron+binary
+    # mech_name = "neurodamus"
```

Comment:
this needs be conditional on neuron than i.e. you have to check spec if it has dependency with `^neuron+cmake`

Reply:
Before we were setting `mech_name` to have a suffix in libnrnmech. 
Without it the lib gonna be named `libnrnmech.so/dylib` which, independently of the neuron version, is always compatible.

Reply:
Ok. May be then we can remove commented lines?

Reply:
In principle all models should override this var (which is set in the parent package to `None`). I like the comment here remembering that the var should stay like that on purpose.